spec/acceptance/types/mysql_login_path_spec.rb

```diff
@@ -6,9 +6,9 @@
 support_bin_dir = '/root/mysql_login_path'
 if os[:family] == 'redhat' && os[:release].to_i == 8
   mysql_version = '8.0'
-elsif os[:family] == 'debian' && os[:release] =~ %r{9|10}
+elsif os[:family] == 'debian' && os[:release] =~ %r{9|10|11}
   mysql_version = '8.0'
-elsif os[:family] == 'ubuntu' && os[:release] =~ %r{16\.04|18\.04}
+elsif os[:family] == 'ubuntu' && os[:release] =~ %r{18\.04|20\.04}
```

Comment:
Should Ubuntu 20.04 be defaulting to mysql 5.7?
Same question for 18.04 tbh

Reply:
Can't really tell. I would expect Ubuntu 20.04 to be able to default to the MySQL 8.0. As for 18.04, someone else listed had it defaulting to 5.7 so I just kept it as is.

Reply:
Huh, look's like it has been set that way since the file was created.......
Weird, but I guess it's ok to leave it as it is for now.